packages/framework/src/schemas/steps/channels/in-app.schema.ts

```diff
@@ -1,10 +1,20 @@
 import { Schema } from '../../../types/schema.types';
 
+const redirectSchema = {
+  type: 'object',
+  properties: {
+    url: { type: 'string' },
+    target: { enum: ['_self', '_blank', '_parent', '_top', '_unfencedTop'] },
```

Comment:
It's a good idea to explicitly declare enum properties with `type` field to ensure the enum values don't regress.

```suggestion
    target: { type: 'string', enum: ['_self', '_blank', '_parent', '_top', '_unfencedTop'] },
```